framework/scripts/find_hung_process.py

```diff
@@ -501,6 +507,8 @@ def main():
     parser.add_argument('-s', '--stacks', metavar='int', action='store', type=int, default=0,
                         help='The number of stack frames to keep and compare for '
                         'uniqueness (Default: ALL)')
+    parser.add_argument('-p', '--param-values', action='store_true', dest="params_are_significant", default=False,
+                        help="Params are significant when comparing stack strings (Default: False)")
```

Comment:
change to:
"Parameters which are significant ........................... (Default: all significant)"

Reply:
what do you mean by parameter?

Reply:
> 13 0x000015202aea0648 in libMesh::PetscVector<double>::init (this=this@entry=foo, n=437434470, n_local=n_local@entry=970200, ghost=..., fast=fast@entry=false, ptype=ptype@entry=libMesh::GHOSTED) at /apps/local/spack/opt/spack/gcc-8.4.1/gcc-9.4.0-57pg55xkr4cydqecxijtmaaptbrgkxcn/include/c++/9.4.0/ext/new_allocator.h:89

See the line above, what I really mean is parameter values are significant (the lhs of those name/value pairs).

Reply:
so 
```
Set whether the argument values for each routine/level in the stack trace are significant for the comparison of stack traces between ranks
```